pkg/shell/shell.scss

```diff
@@ -223,26 +223,10 @@ $desktop: $phone + 1px;
   }
 }
 
-.hostkey-fingerprint {
-  font-size: large;
```

Comment:
There is definitely some UI change here, so I will ask @garrett for review. Personally I think it's fine to drop the override. 

previous:
![Screenshot from 2023-03-21 10-42-28](https://user-images.githubusercontent.com/14921356/226570707-39b00c89-ecbd-4340-9fa0-fb9b3172db88.png)

now:
![Screenshot from 2023-03-21 10-50-42](https://user-images.githubusercontent.com/14921356/226570903-b1f18172-6645-464d-a783-3e33d062e30c.png)